src/olympia/migrations/1001-version-preview.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE `version_previews` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `created` datetime(6) NOT NULL,
+  `modified` datetime(6) NOT NULL,
+  `version_id` int(11) NOT NULL,
+  `position` int(11) NOT NULL,
+  `sizes` longtext NOT NULL,
+  PRIMARY KEY (`id`),
+  KEY `version_previews_version_id_fk_versions_id` (`version_id`),
+  CONSTRAINT `version_previews_version_id_fk_versions_id` FOREIGN KEY (`version_id`) REFERENCES `versions` (`id`)
+);
+CREATE INDEX versionid_position_created ON version_previews(version_id DESC, position, created);
```

Comment:
Okay, tbh though, we could just not have an ordering defined in Meta, nor an index in the database right now, (and drop `position`?) because we're only generating a single Preview per Version, and wanting more than one Version's Previews is an edge case too (or not depending on your thoughts on the `all_previews` below).

If we _do_ end up supporting positions, or multiple Previews per versions, then we might be on django 1.11 by then and we easily add the ordering and/or indexes necessary, and without the extra naming hassles.

Reply:
Mmm, let's drop the `ordering` and position then, yeah.

Reply:
https://github.com/mozilla/addons-server/pull/7766/commits/a9c04ca9f1ea2a9cb81697ccdd3fe2e2e1575529